application/database.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+
+SQLALCHEMY_DATABASE_URL = "sqlite:///./city_temperature_api.db"
```

Comment:
The database URL is hardcoded, which can be a security risk and reduces flexibility. Consider using environment variables to manage the database URL, allowing for different configurations in development and production environments.